Runtimes/Supplemental/Synchronization/CMakeLists.txt

```diff
@@ -0,0 +1,76 @@
+cmake_minimum_required(VERSION 3.29)
+
+set(CMAKE_POSITION_INDEPENDENT_CODE YES) # ToDo: What does this do?
+
+list(APPEND CMAKE_MODULE_PATH "${CMAKE_SOURCE_DIR}/../cmake/modules")
+
+if($ENV{BUILD_NUMBER})
+  math(EXPR BUILD_NUMBER "$ENV{BUILD_NUMBER} % 65535")
+  set(BUILD_NUMBER ".${BUILD_NUMBER}")
+endif()
+project(SwiftSynchronization
+  LANGUAGES Swift
+  VERSION 6.1.0${BUILD_NUMBER})
+
+include(GNUInstallDirs)
+
+if(NOT PROJECT_IS_TOP_LEVEL)
+  message(SEND_ERROR "Swift Synchronization must build as a standalone project")
+endif()
+
+set(${PROJECT_NAME}_SWIFTC_SOURCE_DIR
+  "${PROJECT_SOURCE_DIR}/../../../"
+  CACHE FILEPATH "Path to the root source directory of the Swift compiler")
+
+find_package(SwiftCore)
+include(gyb)
+include(AvailabilityMacros)
+
+add_compile_options(
+  $<$<COMPILE_LANGUAGE:Swift>:-explicit-module-build>
+  $<$<COMPILE_LANGUAGE:Swift>:-enable-builtin-module> # Are these contradictory?
```

Comment:
Which options does the comment refer to? I don't see any contradictory options.

Reply:
@compnerd it was for 

$<$<COMPILE_LANGUAGE:Swift>:-explicit-module-build>
$<$<COMPILE_LANGUAGE:Swift>:-enable-builtin-module>

It was mainly a note for myself to confirm I needed both of those compiler flags as I wasn't sure if they contradicted each other. Evan answered my question below